shared-libs/transitions/src/transitions/utils.js

```diff
@@ -25,8 +25,8 @@ module.exports = {
       errorMessage = messages.getMessage(config, utils.getLocale(doc));
       message = config;
     } else {
-      errorMessage = `messages.generic.${errorKey}`;
-      message = { translation_key: errorMessage };
+      errorMessage = utils.translate(`messages.generic.${errorKey}`, utils.getLocale(doc));
+      message = { translation_key: `messages.generic.${errorKey}` };
```

Comment:
What I think we should do is:
```suggestion
      message = { translation_key: `messages.generic.${errorKey}` };
      errorMessage = messages.getMessage(message, utils.getLocale(doc));
```
and effectively mirror what we do if a config is set. 

Indeed this was a bug from before. Making this change only breaks a couple of tests (one unit and one e2e) for `self_report` which was broken by this as well.